recipes/_websphere_cluster.rb

```diff
@@ -0,0 +1,24 @@
+#
+# Cookbook Name:: websphere
+# recipe:: websphere
+#
+# Copyright (C) 2017 J Sainsburys
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+cookbook_file "#{node['websphere_scripts_location']}/cluster_member_exists.py" do
```

Comment:
we can add this cookbook file to the library `websphere_base` or `websphere_cluster_members`, and not need this recipe at all.